src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -366,7 +367,7 @@ protected function loadFile($file)
         }
 
         try {
-            $configuration = $this->yamlParser->parse(file_get_contents($file));
+            $configuration = $this->yamlParser->parse(file_get_contents($file), Yaml::PARSE_CONSTANT);
```

Comment:
this means that the DI component not can't work with Yaml < 3.2, isn't it? So the composer.json file should get a conflict rule. Or: would it make sense to enable const parsing by default (and thus not require any change here?)